Partial-CI-Tests-Structure.md

```diff
@@ -0,0 +1,17 @@
+## Introduction
+
+If your PR fails due to the `check_ci_test_suites_to_run` workflow, please look at the error log and depending on the error, do the following:
```

Comment:
Add an intro pargraph: Our CI process has a workflow, `check_ci_test_suites_to_run`, that determines which test suites to run for a given PR based on the files that that PR modifies.

Reply:
Done.